Implement Task Data system (#332)
change-metrics/monocle: README.md

docker-compose start crawler
```

### Connect a tasks tracker crawler

The Monocle API provides endpoints for a crawler to send task/issue/RFE
related data into the Monocle database. Monocle is able to attach tasks
to changes based on a match of the `change_url` field.

#### Endpoints available for a crawler

**POST /api/0/task_data?index=<name>&name=<crawler_name>&api_key=<key>**

> This endpoint is protected by an API Key and allow to send a list
> of tasks.

Reviewer: ```suggestion
This endpoint is protected by an API Key and it accepts a list of task data.
```

```Shell
[TASK_DATA]

TASK_DATA: {
  updated_at: str(%Y-%m-%dT%H:%M:%SZ)
  change_url: str
  ttype: List[str]
  tid: str
  url: str
  title: str
  severity: str
  priority: str
  score: str
}
```

**GET /api/0/task_data?index=<name>&name=<crawler_name>**

This endpoint returns the last commit date of a task crawler.

**POST /api/0/task_data/commit?index=<name>&name=<crawler_name>&api_key=<key>**

> str["%Y-%m-%dT%H:%M:%SZ"]
> 
> This endpoint permits a task crawler to set a commit date.

Reviewer: ```suggestion
This endpoint is protected by an API Key and it accepts a timestamp "commit" date to help crawler resume work. 

str["%Y-%m-%dT%H:%M:%SZ"]
```
